lib/fs.js

```diff
@@ -83,7 +83,12 @@ const {
 
 const { FSReqCallback } = binding;
 const { toPathIfFileURL } = require('internal/url');
-const internalUtil = require('internal/util');
+const {
+  customPromisifyArgs,
```

Comment:
```suggestion
  customInspectSymbol: kInspect,
  customPromisifyArgs,
```

Reply:
Does https://github.com/nodejs/node/pull/43159/commits/46d5fdc69baf9ce6209f02b7ff62ee149f979b9e resolve this correctly?

Reply:
Yep, much better than my suggestion would have 😅